policy/policy.go

```diff
@@ -360,7 +360,7 @@ func (p *Policy) updateAllChecksums(ctx context.Context,
 	if p.ScoringSystem == ScoringSystem_SCORING_UNSPECIFIED {
 		p.ScoringSystem = ScoringSystem_AVERAGE
 	}
-	executionChecksum = executionChecksum.Add(p.ScoringSystem.String())
+	executionChecksum = executionChecksum.AddUint(uint64(p.ScoringSystem))
```

Comment:
fyi: we should never depend on the string here; the underlying type is an enum, use it